test/integration/targets/azure_rm_virtualmachine_scaleset/tasks/main.yml

```diff
@@ -28,6 +34,7 @@
     capacity: 2
     virtual_network_name: testVnet
     subnet_name: testSubnet
+    load_balancer: testLB
```

Comment:
Instead of modifying an existing test, we should be creating a new test to specifically try out the new option with 3 tests

- Check mode with assertion that nothing was actually created
- Normal mode with assertion that it was created as specified
- Idempotent with assertion that no changes were recorded or made